app/Http/Controllers/HostedDomainController.php

```
@@ -147,6 +150,60 @@ public function renewSsl(Server $server, Site $site): RedirectResponse
+        $action = app(CheckSslExpiry::class);
+        $checked = 0;
+
+        foreach ($ssls as $ssl) {
+            try {
+                $action->check($ssl, notify: false, ssh: $ssh);
+                $checked++;
+            } catch (Throwable $e) {
+                Log::warning('[SSL expiry check] Failed to check certificate', [
+                    'ssl_id' => $ssl->id,
+                    'error' => $e->getMessage(),
+                ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'checkExpiryAll'. (undefined)

(StaticAccess)